### ISSUE TITLE
GUI: Add Shiny app

### ISSUE THREAD

**Issue by HenrikBengtsson** [MetabolicEngineeringGroupCBMA/seguid#44]
Add Shiny app implementations:

* [ ] **seguid.shiny**: Shiny for R: https://shiny.posit.co/r/getstarted/shiny-basics/lesson1/index.html
* [ ] **seguid.shiny**: Shiny for Python: https://shiny.posit.co/py/

To keep **seguid** lightweight, I propose these are standalone R/Python packages importing **seguid**.  

A user can easily run these locally on their on machines without special privileges.  There are also online services for hosting these, e.g. https://www.shinyapps.io/.  The free account includes 25 hours/month, which should be enough to start with.  It's possible to purchase more hours.

That said, these days, most Shiny apps can also run fully in the web browser via WebAssembly.  There is zero need for a server.  The only thing required is to host static files, which GitHub Pages already supports.  This is supported for both Python and R.  So, it wouldn't take much to make sure these Shiny apps can run standalone in the user's web browser.  From <https://posit-dev.github.io/r-shinylive/>:

* R: https://shinylive.io/r/examples/
* Python: https://shinylive.io/py/examples/

Note, these run completely in your local web browser - no server at all involved.

**Comment by BjornFJohansson** [comment]
Good idea if not to much work. I have an online flask app https://seguidcalculator.pythonanywhere.com/ running on pythonanywhere and this standalone GUI: https://github.com/BjornFJohansson/seguid_calculator?tab=readme-ov-file#seguid_calculator made with wxpython.

**Comment by HenrikBengtsson** [comment]
Related to this: Added an R API online demo based on webR to https://www.seguid.org/demo-webr.html.
